src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/AdditionalConditionUploadDetail.kt

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Lob
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "additional_condition_upload_detail")
+data class AdditionalConditionUploadDetail(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @NotNull
+  val id: Long = -1,
+
+  @NotNull
+  val licenceId: Long,
+
+  @NotNull
+  val additionalConditionId: Long,
```

Comment:
Don't think you need a relationship between the additional_condition table and the licence_table? The licence can already be referenced via the additional condition?

Reply:
This `additional_condition_upload_detail` table has no JPA relationships - it is entirely managed manually, from the earlier discussion about pulling potentially large objects for each licence if we link the detail row in. The licenceId column is purely there for our benefit (for support/visibility) so we can query which licences have an upload detail associated with them. It isn't used in getting data or managing any relationships between tables.